--- om.tmac.orig	2025-03-10 12:43:23.670198728 -0300
+++ om.tmac	2025-03-10 13:39:50.079064365 -0300
@@ -11149,6 +11149,7 @@
 .       while \\n+[@LEVEL]<=\\n[#LOOP] \
 .          if \\n[#HEAD_\\n[@LEVEL]_NUM] \
 .             as $HD_NUM \\n[#HEAD_\\n[@LEVEL]_NUM].
+.       chop $HD_NUM
 .       ie \\n[#TRUNC_TOC_HD_NUM] \
 .          ds $TOC_HD_NUM \\n[#HEAD_\\n[#LEVEL]_NUM].
 .       el .ds $TOC_HD_NUM \\*[$HD_NUM]
@@ -11186,7 +11187,7 @@
 .          as $TOC_HEAD_\\n[#LEVEL]_ITEM "\\*[toc-word-space]
 .    \}
 .    PDF_BOOKMARK \
-        \\*[PDF_NM] \\n[#LEVEL]+1 \
+        \\*[PDF_NM] \\n[#LEVEL] \
         \\*[$PDF_HD_NUM] \\*[$TOC_HEAD_\\n[#LEVEL]_ITEM]
 .    rm PDF_NM
 .    rm $PDF_HD_NUM
@@ -15802,7 +15803,7 @@
 \w'\\*[$TOC_HD_NUM]'u
 .          ti -\w'\\*[$TOC_HD_NUM]'u
 .          pdfhref L -D "\\*[PDFBOOKMARK.NAME]" -E -- \
-\&\\*[$TOC_HD_NUM]\\*[$TOC_HEAD_\\n[#LEVEL]_ITEM]
+\&\\*[$TOC_HD_NUM]\h'\\n[TocDesired]-(\\w'\\*[$TOC_HD_NUM]'/1m)'\\*[$TOC_HEAD_\\n[#LEVEL]_ITEM]
 \!.        ll
 '          in
 .          di TOC_LEADERS
